### PR TITLE
add quote

### PR DESCRIPTION
**Pull request by AARYAN-MAHENDRA**
PR- #13

### CODE REVIEW

File: server/data/quotes.json [hunk 1]
```diff
@@ -1,4 +1,8 @@
 ﻿[
+     {
+    "sentence": "As a hawk cannot fly high with one wing,similarly a man cannot reach excellence with one tongue.",
```
Comment (jayehernandez) [inline server/data/quotes.json:3]: Minor detail, but add a space in between the comma. :)

File: server/data/quotes.json [hunk 2]
```diff
@@ -1,4 +1,8 @@
 ﻿[
+     {
```
Comment (jayehernandez) [inline server/data/quotes.json:2]: Also make this two spaces only.

Comment (AARYAN-MAHENDRA) [reply server/data/quotes.json]: ok

File: server/data/quotes.json (hunk 1 above)
Comment (AARYAN-MAHENDRA) [reply server/data/quotes.json]: ok

**Comment by jayehernandez** [comment]
Thanks for contributing!!

@all-contributors please add @AARYAN-MAHENDRA For content

**Pull request merged**